### PR TITLE
-> [Store config 4 of 4] UsersStore, UserStore, VirtualNetworksStore, VisibilityStore, MesosStateStore

### PR DESCRIPTION
**Pull request by mlunoe**
This PR moves registering of stores their respective constructors instead of it being handled in StoreConfig.

Functionally there is no change from this PR.
Includes the following stores:
- UsersStore
- UserStore
- VirtualNetworksStore
- VisibilityStore
- MesosStateStore

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

**Comment by mlunoe** [comment]
**mesosphere-ci** Retest this please

**Comment by mlunoe** [comment]
This PR:



Master:



Same tests are failing

**Comment by mlunoe** [comment]
This PR fixes the broken tests: https://github.com/dcos/dcos-ui/pull/885

File: src/js/plugin-bridge/PluginSDK.js [hunk 1]
```diff
@@ -15,7 +16,7 @@ const initialState = {};
 const middleware = [ActionsPubSub.pub];
 const PLUGIN_ENV_CACHE = [];
 const REGISTERED_ACTIONS = {};
-const EXISTING_FLUX_STORES = {};
+let existingFluxStores = {};
```
Comment (daiyiwhale) [inline src/js/plugin-bridge/PluginSDK.js]: oh wow nice catch!

**Comment by daiyiwhale** [comment]
Looks good! yay!

File: src/js/plugin-bridge/PluginSDK.js (hunk 1 above)
Comment (philipnrmn) [inline src/js/plugin-bridge/PluginSDK.js]: Roll on immutable.js!

**Comment by philipnrmn** [comment]
I'm a big fan of `tell, don't ask`, I'm a big fan of these PRs.

**Pull request merged**